### PR TITLE
1041 contribution details UI change

### LINKED ISSUE #1041: Contribution details UI change
UI changes for Contribution details page in catalog
Use https://app.zeplin.io/project/62b9dba49dd2b211410e608e/screen/62b9dbf684718712f34fe8d0 as reference

### PR DESCRIPTION
**Pull request by minump**
## Description
Updated layout and styling for contributions details page.
Layout changed with changes in div placement, added rows, columns and table layouts.
Styling changed in style.css classes.
Layout similar to capability and talent details page.

**Fixes #1041**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [x] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: contributions/catalog/webapps/static/css/style.css [hunk 1]
```diff
@@ -527,3 +491,50 @@ input.datadel-api {
 .hidetext { -webkit-text-security: disc; }
 .showtext { -webkit-text-security: none; }
 
+h1 {
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/static/css/style.css:494]: H1 and h4 styles are already added in style.css from line 360. Please remove the duplicates. Thanks.

**Comment by minump** [comment]
converting to draft to work on feedback and discuss potential changes.

**Comment by minump** [comment]
Contribution Admins shown in multiple lines. Need to fix to display in a single line comma separated format.
Screen Shot 2023-01-20 at 12 11 01 PM

"View" buttons for capability view and talent view needs UI change.
"Edit contribution" and "Review contribution" button needs UI change (remove underline, change hover color)

File: contributions/catalog/webapps/static/css/style.css (hunk 1 above)
Comment (minump) [reply contributions/catalog/webapps/static/css/style.css]: updated

Comment (sandeep-ps) [reply contributions/catalog/webapps/static/css/style.css]: Thanks.

**Comment by sandeep-ps** [comment]
I have completed the first review round and updated my suggestions list above.

**Comment by minump** [comment]
> Contribution Admins shown in multiple lines. Need to fix to display in a single line comma separated format. Screen Shot 2023-01-20 at 12 11 01 PM
> 
> "View" buttons for capability view and talent view needs UI change. "Edit contribution" and "Review contribution" button needs UI change (remove underline, change hover color)

Fixed issue.

**Comment by minump** [comment]
> Hi, Minu,
> 
> 1. It would be better if the buttons at the bottom of the page followed the below styles from Zeplin for Edit, Delete, and Review, respectively.
> 
> image
> 
> 2. Currently, the edit and review button texts are showing underlined hover, which needs to be removed.
> 3. Talent and Capability View buttons could also use one of the following button styles:
>    image or image
> 4. The button height dimensions must be fixed for a uniform appearance.
> 5. Could you please check the fonts of the button text against the design? Multiple style elements, including font weight and size, line height, etc., must be corrected in the implementation. Please use Chrome developer tools to confirm the rendered styles with the Zeplin ones.
> 6. The headings on a page need to follow the hierarchy of H1, H2, H3, H4, etc. This is an accessibility requirement. We could talk about this.
> 7. The Review section in the Contribution details UI also needs to be updated.

1. Button styles updated.
2. underline on hover removed.
3. Capability view and Talent "View" buttons styling made similar to "Notification" button.
4. I kinda didnt catch a difference. Maybe we can discuss this.
5. font size and weight updated for button styling.
6. h1 and h4 headings are used in the page. No changes are done yet.
7. Review page is yet to be completed. Reviews.html will be fixed as another PR/issue.
8.

**Comment by minump** [comment]
Ready for review.

**Comment by minump** [comment]
Will work on getting the button heights to match.
The heading tags will be clarified and worked on later.

**Comment by minump** [comment]
Updated button heights, padding and font styles.
Ready for review

File: contributions/catalog/webapps/static/css/style.css [hunk 2]
```diff
@@ -177,83 +136,80 @@ input[type=reset]:hover {
...
-.button3 {
-    color: black;
+.button2, .button2:hover {
+    color: #002855;
     display: inline-block;
     border-radius: 24px;
     border: solid 2px #e84a27;
     background-color: #fff;
+    text-decoration: none;
+    font-weight: 500;
```
Comment (sandeep-ps) [inline contributions/catalog/webapps/static/css/style.css:175]:
```suggestion
    font-weight: bold;
```
Font-weight is marked as bold in Zeplin. Please fix this in this and other buttons too. 
image

Comment (minump) [reply contributions/catalog/webapps/static/css/style.css]: updated font weight for all buttons.

**Review by sandeep-ps — APPROVED** [review]
Thanks for the changes. I will approve this PR. I believe that there may be some scope for improving the implementation. We can plan that as part of a UI implementation fine-tuning task or refactoring in the future.

**Comment by minump** [comment]
Merged latest develop. Ready for review.

**Review by sandeep-ps — APPROVED** [review]

**Comment by minump** [comment]
Merging this PR.

**Pull request merged**